lms/djangoapps/teams/csv.py

```diff
@@ -226,15 +237,18 @@ def load_course_team_memberships(self):
     def load_course_teams(self):
         """
         Caches existing course teams by (team_name, topic_id)
+        and existing membership counts by (topic_id, team_name)
```

Comment:
Out of curiosity, is there a reason for the flipped order?

Reply:
That's just the way the `user_count_by_team` has always been. I don't know why the original author chose to use the different order.